File: src/org/themeswitcher/IdeScheme.java
package org.themeswitcher;

import com.intellij.openapi.editor.colors.EditorColorsManager;
import com.intellij.openapi.editor.colors.EditorColorsScheme;
import org.jetbrains.annotations.Nullable;

class IdeScheme {

    public EditorColorsScheme getGlobalScheme() {
        return EditorColorsManager.getInstance().getGlobalScheme();
    }

    public EditorColorsScheme schemeByName(String scheme) {
        return EditorColorsManager.getInstance().getScheme(scheme);
    }

    public void setGlobalScheme(@Nullable EditorColorsScheme var1) {
        EditorColorsManager.getInstance().setGlobalScheme(var1);
    }
}

File: src/org/themeswitcher/IdeScheme.kt
package org.themeswitcher

import com.intellij.openapi.editor.colors.EditorColorsManager
import com.intellij.openapi.editor.colors.EditorColorsScheme

class IdeScheme {
    var globalScheme: EditorColorsScheme?
        get() = EditorColorsManager.getInstance().globalScheme
        set(value) = EditorColorsManager.getInstance().setGlobalScheme(value)

    fun schemeByName(scheme: String): EditorColorsScheme = EditorColorsManager.getInstance().getScheme(scheme)
}